proto/endpointupdater/endpoint.proto

```diff
@@ -0,0 +1,32 @@
+// Copyright 2021 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+option go_package = "github.com/grpc/test-infra/proto/endpointupdater";
+
+package endpointupdater;
+
+service EndpointUpdater {
+  // Sends an update
+  rpc UpdateEndpoint (EndpointUpdaterRequest) returns (EndpointUpdaterReply) {}
+}
+
+message EndpointUpdaterRequest {
+  string ip_address = 1;
+}
+
+message EndpointUpdaterReply {
+  string error_reason = 1;
```

Comment:
It's more conventional to use [status](https://pkg.go.dev/google.golang.org/grpc/status) to communicate errors.

Reply:
I am not very familiar with how to use the status, can you give me a reference to an example? 

I am assuming that the status is something we can check on the client side when we send out the request, if that is the case, would we be return nothing from the service? Thank you. :)

Reply:
[This post](https://jbrandhorst.com/post/grpc-errors/) looks like a decent write-up on how to use status in Go.

> I am assuming that the status is something we can check on the client side when we send out the request, if that is the case, would we be return nothing from the service

Exactly. Just an empty message.

Reply:
I have read the post, good to learn how to handle gRPC error properly. Updated. Thank you!